commons/src/methods/datetime.ts

```diff
@@ -0,0 +1,51 @@
+export function getTimeString(date: Date): string {
+  // 11:11am
+  return date
+    .toLocaleTimeString([], { timeStyle: "short" })
+    .replaceAll(/\./g, "");
```

Comment:
Fix for above is to upgrade typescript https://github.com/microsoft/TypeScript/issues/38266#issuecomment-912508678